src/app/downloads/downloads.component.ts

```diff
@@ -126,12 +127,17 @@ export class DownloadsComponent implements OnInit, OnDestroy {
   }
 
   setDefaultEpisodeFilter() {
-    if (!this.filter.episodes ||
-        this.filter.episodes.length < 5) {
-      // init with the first five (or less) episodes
-      const episodes = this.allPodcastEpisodes.length > 5 ? this.allPodcastEpisodes.slice(0, 5) : this.allPodcastEpisodes;
-      this.store.dispatch(new CastleFilterAction({filter: {episodes}}));
-    }
+    const episodes = [];
+    this.allPodcastEpisodes.every((episode: EpisodeModel) => {
+      episodes.push(episode);
+      if ((episode.publishedAt.valueOf() < this.filter.beginDate.valueOf() &&
+          episodes.length % 5 === 0) || episodes.length === DownloadsComponent.DONT_BREAK_CASTLE_LIMIT) {
```

Comment:
_(edge case)_ Does this mean if you chose a window containing no episodes (like "10 years ago"), it would still load 5 of them from Castle?

Reply:
hmm, yes. This is not ideal, but it is kind of an edge case to get it to load like this... I'm thinking if this default episode filter sticks, it could shift with the user selected dates. I feel like there will be more adjustments made here.
<img width="643" alt="screen shot 2017-10-25 at 6 41 32 pm" src="https://user-images.githubusercontent.com/2250413/32028436-376b7a68-b9b4-11e7-968f-5e9102e0aec1.png">